src/app_engine/apprtc.py

```diff
@@ -109,26 +66,30 @@ def get(self, room_id):
         self.write_response('full_template.html')
         return
     # Parse out room parameters from request.
-    params = parameter_handling.get_room_parameters(self.request, room_id, None, None)
+    params = parameter_handling.get_room_parameters(self.request, room_id,
+                                                    None, None)
     # room_id/room_link will be included in the returned parameters
     # so the client will launch the requested room.
     self.write_response('index_template.html', params)
 
+
 class ParamsPage(webapp2.RequestHandler):
   def get(self):
     # Return room independent room parameters.
-    params = parameter_handling.get_room_parameters(self.request, None, None, None)
+    params = parameter_handling.get_room_parameters(self.request,
+                                                    None, None, None)
     self.response.write(json.dumps(params))
 
 app = webapp2.WSGIApplication([
     ('/', MainPage),
-    ('/decline/(\w+)', decline_page.DeclinePage),
-    ('/join/(\w+)', join_page.JoinPage),
-    ('/leave/(\w+)/(\w+)', LeavePage),
-    ('/message/(\w+)/(\w+)', MessagePage),
-    ('/params', ParamsPage),
-    ('/register/(\w+)', gcm_register.BindPage),
-    ('/r/(\w+)', RoomPage),
+    (r'/decline/(\w+)', decline_page.DeclinePage),
```

Comment:
raw string. We were lucky because w isn't escaped.
so e.g. 'a\nb' vs r'a\nb' first one has newline second one doesn't